doc/classes/Camera2D.xml

```diff
@@ -346,16 +352,21 @@
 		<member name="editor_draw_screen" type="bool" setter="set_screen_drawing_enabled" getter="is_screen_drawing_enabled">
 		</member>
 		<member name="limit_bottom" type="int" setter="set_limit" getter="get_limit">
+			Bottom limit the camera will be able to move in pixels.
 		</member>
 		<member name="limit_left" type="int" setter="set_limit" getter="get_limit">
+			Left limit the camera will be able to move in pixels.
 		</member>
 		<member name="limit_right" type="int" setter="set_limit" getter="get_limit">
+			Right limit the camera will be able to move in pixels.
 		</member>
 		<member name="limit_smoothed" type="bool" setter="set_limit_smoothing_enabled" getter="is_limit_smoothing_enabled">
 		</member>
 		<member name="limit_top" type="int" setter="set_limit" getter="get_limit">
+			Top limit the camera will be able to move in pixels.
 		</member>
 		<member name="offset" type="Vector2" setter="set_offset" getter="get_offset">
+			The scroll offset, useful for looking around or camera shake animations
```

Comment:
This is more accurately the camera offset, not scroll offset (it applies even if the camera doesn't scroll). Also missing period at end of sentence.